api/api.go

```diff
@@ -92,11 +94,15 @@ func New(
 	}
 
 	handler := handlers.CompressHandler(router)
+	if isReqLoggerEnabled {
+		handler = RequestLoggerHandler(handler, log15.New("pkg", "api"))
```

Comment:
`log15.New("pkg", "api")` should be a package level global variable

Reply:
yeah fair point.